blog/2025-04-03-mcp-server/index.mdx

```diff
@@ -160,6 +161,79 @@ The Model Context Protocol server will run in the same level of sandboxing as
 your editor does. Be careful with what you install and always double-check what
 you run before you run it.
 
+In order to make this as safe as possible, we’ve made the Model Context Protocol
+also available as a docker container. This means you can run it in a sandboxed
+environment and not have to worry about it having access to your local
+filesystem. You can run it in a container and container have access to a
+specific directory on your local filesystem. This is a great way to make sure
+that the Model Context Protocol server can only access the files you want it to.
+
+<Tabs>
```

Comment:
The folks who are going to use the MCP most likely won't be willing to work with complex setup instructions. We need to set this up to be as much one-click / one-command as possible. And having to build a docker image introduces friction.

Reply:
Let's do that and update the instructions. We have a `tigrisdata` org on dockerhub where a new repository can be created and image uploaded.